src/linked-list.js

```diff
@@ -1,9 +1,64 @@
+// * Should have the methods: `addToTail`, `removeHead`, and `contains`.
+// * `addToTail` replaces the tail with a new value that is passed in.
+// * `removeHead` removes and returns the head node.
+// * `contains` should searth through the linked list and return true if
+// * a matching value is found.
+// * The `head` property is a reference to the first node and the `tail`
+// * property is a reference to the last node.  These are the only two properties
+// * that you need to keep track of an infinite number of nodes.  Build your nodes with objects.
+
+class Node {
+  constructor(element) {
+    this.value = element;
+    this.next = null;
+    this.prev = null;
+  }
+}
 class LinkedList {
   constructor() {
     this.head = null;
     this.tail = null;
-    // Do not modify anything inside of the constructor
+   // Do not modify anything inside of the constructor
+  }
+  addToTail(element) {
+    const currentNode = new Node(element);
+    if (!this.head) this.head = currentNode;
+    else {
+      this.tail.next = currentNode;
+      currentNode.prev = this.tail;
+    }
+    this.tail = currentNode;
+  }
+  removeHead() {
+    if (!this.head) return;
```

Comment:
Instead of just returning, perhaps return `undefined` or `null` to indicate that the function ran, but that there was no head to remove. If nothing is returned, then from a user's perspective, they may not be able to immediately realize that the reason nothing was returned was that there was no head.